app/src/features/Mappings/FhirProfileStep.tsx

```diff
@@ -58,11 +58,10 @@ const FhirProfileStep = ({
   const { t } = useTranslation();
   const classes = useStyles();
   const [isDefaultProfileSelected, setDefaultProfileSelected] = useState(false);
-  const { definition_id: resource } = mapping;
 
   const handleClickDefaultProfile = () => {
     setDefaultProfileSelected(true);
-    onChange && onChange({});
+    onChange && onChange();
```

Comment:
I guess this will be usefull when we plug the profile features

Reply:
Nvm, if the design stays the same as other `Step` components, I guess it'll dispatch actions instead of using `onChange` callback